employee-onboarding-A2A-reference-implementation/src/auth/jwt_validator.py

```
@@ -0,0 +1,198 @@
+    """Validates JWTs against Asgardeo JWKS."""
+    
+    def __init__(self):
+        self.settings = get_settings()
+        self._jwks: Optional[dict] = None
+        self._http_client: Optional[httpx.AsyncClient] = None
+    
+    @property
+    def http_client(self) -> httpx.AsyncClient:
+        if self._http_client is None:
+            # Disable SSL verification for local dev (localhost:9443)
+            self._http_client = httpx.AsyncClient(timeout=30.0, verify=False)
```

Ruff (0.15.2)

[error] Probable use of `httpx` call with `verify=False` disabling SSL certificate checks

(S501)